source/code/plugins/in_vminsights.rb

```diff
@@ -21,29 +21,30 @@ def configure(conf)
         super
 
         begin
+            @log = LogWrapper.new(LogPrefix, @log)# unless @log.kind_of? LockWrapper
```

Comment:
Do you want to keep the commented out part?

Reply:
Ooops!